data/hai missions.txt

```diff
@@ -1053,3 +1053,245 @@ mission "Hai Honeymoon"
+			`	"Look kid, Unfettered attacks make it so they need a lot of medical supplies in Cloudfire, and this here is the best medical equipment Hai space has to offer, these few crates could save many military lives and I don't think I should be trusting something like this to a beginner." he says.`
+			`	If you were the client for this job you certainly wouldn't want a beginner going through such a dangerous territory with such valuable cargo, but he seems desperate, and seeing as the spaceport is deserted he doesn't have many other captains to look for help, perhaps you could convince him.`
+			choice
+				`	"Beginner or not, I'll be escorting her there. I'm pretty confident in my skills and my ship is nothing to scoff at, so I'll keep her, and the cargo, safe."`
+					goto safe
+				`	"Don't worry, the planet is one jump away, what could possibly go wrong?"`
+					goto wrong
+				`	(Make up stories about Nanachi's experience.)`
+					goto lie
+			label safe
+			`	He seems a bit surprised by your tone, but responds positively. "Well, yes of course you look like a very experienced captain. Sorry if I sounded a bit rude before, I'm doing repairs, and have gotten really stressed, you know how it is. I'm sure you can handle it, just be careful."`
+				accept
+			label wrong
+			`	"She could be blasted to bits in an instant by an Unfettered fleet is what could go wrong, dying because of stupid recklesness!" He sighs and looks back at the crates, then takes a while looking around the deserted spaceport, as if trying to find some other captain, until he finally turns to Nanachi. "But I guess beggars can't be choosers. Fine, I'll let you take the cargo there, but be careful! And just so we're clear, you're only getting a small cut of the pay, I haven't been flying like a maniac from one end of Hai space to the other to get nothing!"`
+				accept
+			label lie
+			`	You go on about how Nanachi was the one who taught you how to fly a ship, and how famous she is amongst the Hai pilots, but as he looks at you with great disdain, he interrupts your stories. "Oh of course, an experienced captain that has been running around the spaceport for hours and can't even figure out the job board! I felt like I could maybe trust you to take care of her and the cargo, buy if you won't at least be honest with a fellow captain I can't put you in charge of a mission like this. Now get out of here, I have better things to do than waste my time listening to nonsense!" You and Nanachi continue to look for someone willing to give her a job, but after hours it's clear your best bet was with that man. Unable to find any other way to help her, you apologize and wish her the best of luck.`
+				decline
+	npc accompany save
+		personality escort timid
+		government "Hai"
+		ship "Heavy Shuttle" "Acorn Chaser"
+	on visit
+		dialog `	You have reached <planet>, but you left Nanachi's ship behind! Better depart and wait for them to arrive in this star system.`
```

Comment:
`dialog` follows the same rules as `conversation`: the first one isn't tabbed in.
```suggestion
		dialog `You have reached <planet>, but you left Nanachi's ship behind! Better depart and wait for them to arrive in this star system.`
```